Added Packer v1.16.x Release docs (#2841)
hashicorp/web-unified-docs: content/packer/v1.16.x/content/docs/commands/verify-attestation.mdx

---
page_title: packer verify-attestation command reference
description: The `packer verify-attestation` command verifies a signed DSSE attestation and can enforce provenance policy checks.
---

# `packer verify-attestation` command

The `packer verify-attestation` command verifies a signed DSSE attestation. You
can also require an expected predicate type, builder identity, source URI, or
artifact digest.

## Usage

```shell
packer verify-attestation [OPTIONS] ATTESTATION
```

> `ATTESTATION` is the path to the DSSE attestation file.

Reviewer: ```suggestion
`ATTESTATION` must be a path to a DSSE attestation file.
```

## Options

- `-signing-mode` - Signing mode: `key`, `kms`, or `keyless`. Packer detects the mode when possible.

> - `-key` - PEM key path or KMS or Vault URI.

Reviewer: ```suggestion
- `-key` - PEM key path, KMS URI, or Vault URI.
```
